WebApp/handler/SystemAdministrationHandler.py

```diff
@@ -581,7 +581,7 @@ def sendMail(self, sendDict=None, title=None, body=None, fromAddress=None):
     sentSuccess = list()
     sentFailed = list()
     gLogger.debug("Initializing Notification client")
-    ntc = NotificationClient(lambda x, timeout: RPCClient(x, timeout=timeout, static=True))
```

Comment:
The NotificationClient no longer takes a functor
https://github.com/DIRACGrid/DIRAC/commit/f719136fd27e6fb7063e8aa1a44ba500e6175207